shell/linters/bash.sh

```diff
@@ -7,21 +7,23 @@ SHELLCHECKPATH="$DIR/shellcheck.sh"
 # shellcheck disable=SC2034
 extensions=(sh bash)
 
-shellcheck() {
-  if ! git ls-files '*.sh' | xargs -n40 "$SHELLCHECKPATH" -x -P SCRIPTDIR; then
-    error "shellcheck failed on some files. Run 'make fmt' to fix."
```

Comment:
I actually really liked having these errors come up. Could we either leave them in or add a top level error like: `Linting faild on some files. Run 'make fmt' to fix.`